README.md

```diff
@@ -2,5 +2,435 @@
 ==========
 [![Build Status](https://github.com/razz-team/eva/actions/workflows/main_build.yml/badge.svg)](https://github.com/razz-team/eva/actions)
 [<img src="https://img.shields.io/maven-central/v/team.razz.eva/eva-domain.svg?label=latest%20release"/>](https://search.maven.org/search?q=g:team.razz.eva%20OR%20g:team.razz.eva)
+
+Welcome to Eva! It is a Kotlin open-source framework, which helps you to write your code in DDD style and using CQRS approach.
+
+
+## Getting started
+
+Declare Eva dependencies in your project:
+
+```kotlin
+dependencies {
+    implementation("team.razz.eva:eva-uow:$eva_version")
+    implementation("team.razz.eva:eva-repository:$eva_version")
+    implementation("team.razz.eva:eva-persistence-jdbc:$eva_version")
+}
+```
+
+### Model
+Define events for your business domain.
```

Comment:
We can think about `object NoopEvent<Id> : ModelEvent<Id>` and `class ModelWithoutEvent<Id, NoopEvent<Id>>` for potential users who not interested in event sourcing.